tests/RME/gold_hrrr/config.ini

```diff
@@ -294,7 +295,7 @@ temp:                          -2.5
 ################################################################################
 
 [output]
-out_location:                  /home/micahsandusky/Code/smrf/tests/RME/output
+out_location:                  /home/scotthavens/code/smrf/tests/RME/output
```

Comment:
Lets changes the paths back to relative to avoid user names.

Reply:
Removing